bin.js

```diff
@@ -5,7 +5,6 @@ const fs = require('fs')
 const path = require('path')
 const opn = require('opn')
 const ora = require('ora')
-const async = require('async')
```

Comment:
are we using `async` in some other place? Should we remove it from our deps?

Reply:
It's still in use in `lib/clean.js`